geode/models/distance_matrix.py

```diff
@@ -82,14 +83,14 @@ def __get__(self, instance, owner):
 
         subresults = await asyncio.gather(*[
             self.fn(instance, origins=origins[y:y2 + 1], destinations=destinations[x:x2 + 1], *args, **kwargs)
-            for x, y, x2, y2 in chunks])
+            for x, y, x2, y2 in chunks
+        ])
 
         for subres, chunk in zip(subresults, chunks):
             x, y, x2, y2 = chunk
-            xs = x2 - x + 1
-            ys = y2 - y + 1
 
-            results[y:ys, x:xs] = subres.distances
+            if subres.distances.size:
+                results[y:y2+1, x:x2+1] = subres.distances
```

Comment:
fix to array indexing

Reply:
Off by 1 error?

Reply:
no this was just wrong, offset vs indices error.
[1:1, 2:1] are instead now [1:2, 2:3]